worldtimebuddy/utils.py

```diff
@@ -1,9 +1,24 @@
 import os
 from typing import List
+import datetime
 
 def get_major_tz_from_env() -> List[str]:
     try:
         major_time_zones = os.environ.get("MAJOR_TIMEZONES", None)
         return list(major_time_zones[1:len(major_time_zones) - 1].split(','))
     except:
-        return []
\ No newline at end of file
+        return []
+
+def callDelta(time: datetime.datetime, delta: str) -> datetime.datetime:
+    if 'hr' in delta:
+        hours = int(delta.replace('hr', ''))
+        time += datetime.timedelta(hours=hours)
+    elif 'min' in delta:
+        minutes = int(delta.replace('min', ''))
+        time += datetime.timedelta(minutes=minutes)
+    elif 'day' in delta:
+        days = int(delta.replace('day', ''))
+        time += datetime.timedelta(days=days)
+    else:
+        raise ValueError("Invalid time format. Use +2hr, -30min, -1day, etc.")
```

Comment:
If returning a datetime how will the ValueError behave? Can you think of better approach here for return type? Also please attach testing for negative case as well where explicitly the else case gets executed.

Reply:
> If returning a datetime how will the ValueError behave? Can you think of better approach here for return type? 

should I define a custom error in case an invalid argument is passed and else case is executed?

Reply:
I think ValueError seems fine, but in that case there is no return value right? How are we handling that?

Reply:
> If returning a datetime how will the ValueError behave? Can you think of better approach here for return type?

when ValueError is raised its caught by the try catch block in cli.py which calls the delta function, I can rename it to a custom error to avoid confusion, will that do?

Reply:
Yes, also update the testing for that.